odml/base.py

```diff
@@ -2,12 +2,15 @@
 """
 Collects common base functionality
 """
-import collections
 import posixpath
 
 from . import terminology
 from .tools.doc_inherit import allow_inherit_docstring
 
+try:
+    from collections.abc import Iterable
+except:
```

Comment:
Can you please replace `except` with `except ImportError`

This would maintain uniformity and help us make sure that we are not skipping any other unwanted exception alongside as well.

Reply:
Thanks for the catch. I missed it. Should be fixed now.